lib/apiservers/portlayer/swagger.json

```diff
@@ -1748,6 +1748,63 @@
 				}
 			}
 		},
+		"/containers/{handle}/name": {
+			"put": {
```

Comment:
@vburenin I discussed with Hasan and decided to use put instead of post. What do you think @hmahmood